examples/demo.rs

```diff
@@ -5,37 +5,45 @@ struct Stage {
     show_egui_demo_windows: bool,
     egui_demo_windows: egui_demo_lib::DemoWindows,
     color_test: egui_demo_lib::ColorTest,
-    pixels_per_point: f32,
+    zoom_factor: f32,
+    mq_ctx: Box<dyn mq::RenderingBackend>,
 }
 
 impl Stage {
-    fn new(ctx: &mut mq::Context) -> Self {
+    fn new() -> Self {
+        let mut mq_ctx = mq::window::new_rendering_backend();
+
         Self {
-            egui_mq: egui_mq::EguiMq::new(ctx),
+            egui_mq: egui_mq::EguiMq::new(&mut *mq_ctx),
             show_egui_demo_windows: true,
             egui_demo_windows: Default::default(),
             color_test: Default::default(),
-            pixels_per_point: ctx.dpi_scale(),
+            zoom_factor: mq::window::dpi_scale(),
```

Comment:
This is wrong - the default zoom factor in egui is always 1.0, but the default dpi-scale could be e.g. 2.0 for a retina screen.

But I suggest we just remove the `zoom_factor` stuff from the demo, since this is now built-in to egui with `ctrl+/0`.

Reply:
Whoops, fixed.

I'll let @not-fl3 decide whether to remove the zoom factor stuff - I'm just preserving the existing functionality here.